scripts/katana_version/__init__.py

```diff
@@ -1,14 +1,16 @@
 import logging
+from argparse import Namespace
```

Comment:
aside: should we enable `check_python_format.sh` for scripts or at least `scripts/katana_version/`?

Reply:
Yes. I already have that running locally, hence these changes (I guess I must have written this code before that was setup). Separate PR obviously.

Reply:
https://katanagraph.atlassian.net/browse/ENG-274